docs/how-to-guides/develop-a-flex-flow/function-based-flow.md

```diff
@@ -93,7 +93,8 @@ A flow YAML may look like this:
 $schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
 entry: path.to.module:function_name
 sample:
-  question: "what's the capital of France?"
```

Comment:
I have a concern, will any users use it this way before? And our latest version of promptflow doesn't support this?

Reply:
Good catch. Seems other docs' sample field doesn't have inputs. Let me remove it to reduce confusion

Reply:
Synced with @wangchao1230 , according to current implementation, flows will only support sample with inputs. Prompty will support sample without inputs.